src/native/corehost/hostmisc/pal.windows.cpp

```diff
@@ -434,7 +434,7 @@ pal::string_t pal::get_current_os_rid_platform()
     // Init the buffer
     ZeroMemory(&osinfo, sizeof(osinfo));
     osinfo.dwOSVersionInfoSize = sizeof(osinfo);
-    HMODULE hmodNtdll = LoadLibrary("ntdll.dll");
+    HMODULE hmodNtdll = LoadLibraryA("ntdll.dll");
```

Comment:
Was this change needed?

Reply:
coreclr defines LoadLibrary as LoadLibraryW. Without the change I would have to change the string literal to `_X("ntdll.dll")`, but then the standalone hosts would not build because they map LoadLibrary to LoadLibraryA.

Since we use a char string here, I assumed we expect LoadLibraryA to be called, so I made it more explicit.

---
In reply to: [578794607](https://github.com/dotnet/runtime/pull/48254#discussion_r578794607) [](ancestors = 578794607)